### ISSUE TITLE
Error: Cannot find module 'config/tailwind'

### ISSUE THREAD

**Issue by zlpkhr** [formaat-design/community#148]
Error happens while following tailwind integration tutorial. I found that `tailwind.js` file in root imports from `config/tailwind`, while `postcss.js` uses relative import from `./config/postcss`. Temporally, importing `getTheme` directly from `reshaped/config/tailwind` fixes the issue

```js
// node_modules/reshaped/postcss.js
const config = require("./config/postcss");

module.exports = { config };
```

```js
// node_modules/reshaped/tailwind.js
const tailwind = require("config/tailwind");

module.exports = tailwind;
```

```js
// Temporary fix
// tailwind.config.js
const { getTheme } = require("reshaped/config/tailwind");

/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./pages/**/*.{js,ts,jsx,tsx}"],
  theme: getTheme(),
};
```

**Comment by blvdmitry** [comment]
Thanks for the heads-up, we have this reported and added into our backlog and I'm planning to release a patch this week

**Comment by blvdmitry** [comment]
I've just published 1.14.1 with the fix to the import path and checked it in the example folder: https://reshaped.so/content/changelog

Let me know if it works fine now or if it's still happening for you

**Comment by zlpkhr** [comment]
Works great, thank you!
